src/Handler/UdpHandler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\VarDumper\Handler;
+
+use Exception;
+use RuntimeException;
+use Socket;
+use Yiisoft\VarDumper\HandlerInterface;
+
+final class UdpHandler implements HandlerInterface
+{
+    private ?Socket $socket = null;
+
+    public function __construct(
+        private string $host,
+        private int $port,
+    ) {
+        if (!extension_loaded('sockets')) {
```

Comment:
It's not necessary to use the `socket` extension. You can use `fpsockopen` and `fwrite` functions instead. And don't forget to close connections.